### ISSUE TITLE
Improve flexibility in what is updated

### ISSUE THREAD

**Issue by Steve-Mcl** [leftstick/vscode-npm-dependency#8]
Hi, i wonder if you would consider providing greater flexibility of what is updated (and to what version) by permitting users to update what is highlighted? 

Example, something like this...

![image](https://user-images.githubusercontent.com/44235289/90606422-bc448400-e1f7-11ea-99b2-8c05f852bdf0.png)

Thanks for the consideration.

**Comment by leftstick** [comment]
it is now supported via config. 

So you are asking if it is possible to make it easier to use(all options in right context menu) ?  I think it is a nice idea. 

@geforcesong  would you like to check how we can improve it?

**Comment by Steve-Mcl** [comment]
Yes, asking for the options to be available via pallete menu and context menu. 

But more importantly, only update selected/highlighted items. 

There are often registered items I DONT want to update or perhaps there is only one item I want to update to beta for testing purposes. So operating the updates on only highlighted items is very very useful. 

Thanks again for the consideration. 

Steve.

**Comment by leftstick** [comment]
`only update selected/highlighted items.`

this seems nice feature, thanks for your suggestion
